rpcs3/rpcs3qt/gs_frame.cpp

```diff
@@ -353,7 +353,22 @@ void gs_frame::keyPressEvent(QKeyEvent *keyEvent)
 				gui_log.warning("The video provider could not release the image sink. A sink with higher priority must have been set.");
 			}
 
-			rsx::overlays::queue_message(tr("Recording stopped: %0").arg(QString::fromStdString(m_video_encoder->path())).toStdString());
+			// Play a sound
+			Emu.CallFromMainThread([]()
```

Comment:
There is no need to call this from the mainthread, as we already are in the main thread

Reply:
> There is no need to call this from the mainthread, as we already are in the main thread

Again, it is ported from the screenshot function.
I can of course modify both of them if you wish.

Reply:
Done.